onnxscript/backend/onnx_export.py

```diff
@@ -199,14 +182,76 @@ def _attribute_value(attr: onnx.AttributeProto):
     raise NotImplementedError(f"Unable to return a value for attribute {attr!r}.")
 
 
+def _update_names_used_in_graph(names: set[str], graph: GraphProto) -> None:
+    """Returns the names used in a graph."""
```

Comment:
This returns None. Maybe "Adds the named used in a graph to the set names." ?

Reply:
Done